src/esm_runscripts/namelists.py

```diff
@@ -346,7 +346,7 @@ def echam_transient_forcing(config):
 
                 try:
                     forcing_table = pd.read_csv(
-                        config["echam"]["transient_forcing_table"], sep=";", index_col=0
+                        config["echam"]["transient_forcing_table"], sep=";", index_col=0, header=None, 
```

Comment:
Just out of curiosity, why did you need the `header=None`?

Reply:
Just out of convenience for me. In my opinion, it is less ambiguous, but I am totally fine with not setting `header`. I don't know how many people are using this though

Reply:
I at some point wrote the transient forcing table function. `header=None` is just something you give to pandas to ensure that it does not use the first row as a label for the rows. The default is "infer" (https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html), and explicitly setting `None` seems to be safer (in my opinion)

Reply:
thanks for the clarifications guys!